static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -568,10 +567,7 @@ export const getEpicTestsFromGoogleDoc = (): Promise<
                                 ','
                             ),
                             copy: {
-                                heading: throwIfEmptyString(
-                                    'heading',
-                                    row.heading
-                                ),
+                                heading: row.heading,
```

Comment:
In [the template](https://github.com/guardian/frontend/blob/9a414904ba2bb13fdd7707ac3d26bd11f7b12092/static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-epic-control.js#L27) you provide a default in the template of empty string but don't remove the `<h2>` so you end up with an empty element.